src/main/java/org/spongepowered/common/world/gen/SpongeWorldGenerator.java

```diff
@@ -124,7 +120,7 @@ public GenerationPopulator getBaseGenerationPopulator() {
 
     @Override
     public void setBaseGenerationPopulator(GenerationPopulator generator) {
-        this.baseGenerator = checkNotNull(generator, "generator");
+        this.baseGenerator = checkNotNull(generator);
```

Comment:
Why remove the second param? It's useful when you actually encounter this exception (if you do).